test/cmdlineTests/name_simplifier/output

```diff
@@ -58,8 +58,12 @@ object "C_56" {
             function abi_decode_t_struct$_S(headStart, end) -> value
             {
                 if slt(sub(end, headStart), 0x20) { revert(value, value) }
-                value := allocateMemory(0x20)
-                mstore(value, calldataload(headStart))
+                let memPtr := mload(64)
```

Comment:
Why did the revert -> invalid change influence these other changes?

Reply:
A new function was added and that changes the way the inliner works.

Reply:
Ah and `allocateMemory` now has `invalid()` instead of `revert(0, 0)` and thus the function is smaller and can be inlined.

Reply:
Right